src/vso-task-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-task-lib",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
you also need to change the vsoagent package.json to conume this 0.3.0 right?  It's in the root of the agent repo